pkg/alerts/alertsHandler/alertsHandler.go

```diff
@@ -173,6 +181,115 @@ func ProcessSilenceAlertRequest(ctx *fasthttp.RequestCtx) {
 	utils.WriteJsonResponse(ctx, responseBody)
 }
 
+func ProcessTestContactPointRequest(ctx *fasthttp.RequestCtx) {
+	var testContactRequest TestContactPointRequest
+	if err := json.Unmarshal(ctx.PostBody(), &testContactRequest); err != nil {
```

Comment:
you should check if the PostBody is not nil/empty, else it might panic (verify via postman what happens)

Reply:
if the postbody is nil, `json.Unmarshal` returns an error and the API returns 

```
{
    "error": "Failed to unmarshal json"
}
```
Just verified this on postman